store/postgres/src/query_store.rs

```diff
@@ -85,11 +85,11 @@ impl QueryStoreTrait for QueryStore {
         self.store.wait_stats(self.replica_id)
     }
 
-    async fn has_non_fatal_errors(&self, block: Option<BlockNumber>) -> Result<bool, StoreError> {
+    async fn has_non_fatal_errors(&self, block: BlockNumber) -> Result<bool, StoreError> {
```

Comment:
To be consistent this should also be renamed to `has_deterministic_errors`.

Reply:
Fixed!